src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,17 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+    private static final int MAX_RADIUS_NUMBER = 10;
+    private final Random random = new Random();
+    private final ColorSupplier supplier = new ColorSupplier();
+
+    public Figure getRandomFigure() {
+        return null;
+    }
+
+    public Figure getDefaultFigure() {
+        return new Circle(supplier.getRandomColor(), random.nextInt(MAX_RADIUS_NUMBER));
```

Comment:
Avoid passing expressions directly into constructors for better code readability. Create separate variables for the color and radius, and then pass them to the `Circle` constructor.